mkosi/config.py

```diff
@@ -1860,6 +1860,9 @@ class Config:
     kernel_modules_initrd_exclude: list[str]
     kernel_modules_initrd_include_host: bool
 
```

Comment:
Please drop this newline to keep it together with the kernel module include/exclude settings

Reply:
There is an empty line a bit before, should I remove that too?

https://github.com/systemd/mkosi/blob/c2b7730ea0246285149d924c6d467808981ded2e/mkosi/config.py#L1853-L1861

Reply:
Keep it separate from the initrd options but next to the regular include/exclude options. Maybe github messed up the formatting